.travis.yml

```diff
@@ -7,4 +7,9 @@ go:
   - 1.7.x
   - 1.8.x
   - 1.9.x
+  - "1.10.x"
```

Comment:
Do you want to go ahead and make these consistent?

Reply:
@domesticmouse 
Do u mean adding a double quote ?
I do not get it.